tools/generator/src/Generator/Main.swift

```diff
@@ -47,7 +47,7 @@ extension Generator {
     }
 
     static func parseArguments(_ arguments: [String]) throws -> Arguments {
-        guard CommandLine.arguments.count == 8 else {
+        guard arguments.count == 8 else {
```

Comment:
Lets replace it everywhere in this functions, not just here.

Reply:
Done